src/modules/land_detector/VtolLandDetector.h

```diff
@@ -59,21 +59,23 @@ class VtolLandDetector : public MulticopterLandDetector
 	void _update_topics() override;
 	bool _get_landed_state() override;
 	bool _get_maybe_landed_state() override;
+	bool _get_freefall_state() override;
 
 private:
 
-	uORB::Subscription _airspeed_sub{ORB_ID(airspeed)};
+	uORB::Subscription _airspeed_validated_sub{ORB_ID(airspeed_validated)};
 	uORB::Subscription _vehicle_status_sub{ORB_ID(vehicle_status)};
 
-	airspeed_s _airspeed{};
+	airspeed_validated_s _airspeed_validated{};
 	vehicle_status_s _vehicle_status{};
 
 	bool _was_in_air{false}; /**< indicates whether the vehicle was in the air in the previous iteration */
 	float _airspeed_filtered{0.0f}; /**< low pass filtered airspeed */
 
 	DEFINE_PARAMETERS_CUSTOM_PARENT(
 		MulticopterLandDetector,
-		(ParamFloat<px4::params::LNDFW_AIRSPD_MAX>) _param_lndfw_airspd_max
+		(ParamFloat<px4::params::LNDFW_AIRSPD_MAX>) _param_lndfw_airspd_max,
+		(ParamFloat<px4::params::ASPD_STALL>)  	    _param_aspd_stall
```

Comment:
@dagar We have a bit of a problem with this parameter as it's defined as part of the airspeed module which is not built for all targets. This header however is always included so some builds will fail. Any suggestions from your side?